### ISSUE TITLE
Coroner (has TRAIT_MORBID) doesn't actually get morbid mood buffs/debuffs

### ISSUE THREAD

**Issue by Metekillot** [tgstation/tgstation#86919]
### Client Version:

515.1642

### Issue Summary:

The title captures it. I'm just making this issue so I don't forget to fix it at some point, unless someone else wants to take the GBP.

### Round ID:

239040

### Test Merge Information:

- [[TM ONLY] Hijacked Revision Date for v3 TGS Api](https://www.github.com/tgstation/tgstation/pull/83227)
- [Adds new station map Nebulastation](https://www.github.com/tgstation/tgstation/pull/84826)
- [Adds a warning and single retry to byond membership lookup](https://www.github.com/tgstation/tgstation/pull/86765)
- [Adds the NT BR-38 Battle Rifle. A sci-fi hybrid gun (that can critically fail on existence). Throws combat shotguns (the big ones from cargo) into the bin. Other changes. (SPRITES WIP)](https://www.github.com/tgstation/tgstation/pull/86853)

### Reproduction Steps:

Coroner (or anyone with TRAIT_MORBID) is meant to get mood buffs for doing fucked up surgeries, and debuffs for saving lives and things like that. Try it out. You just get the normal mood things.

**Comment by 00-Steven** [comment]
hrmn, failing to reproduce this so far with organ manipulation, revival surgery, autopsy, and tending wounds

**Comment by TheVekter** [comment]
Yeah, I haven't played in the last week or two but last time I tried, I was getting the moodlets correctly. What exactly were you doing? Were you using the morbid tools?
